spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] Hi @Paulinelali, based on the requirements of this project, any custom methods in the models should have a corresponding unit test. Let's kindly add unit test for the method `#top_three_posts` :+1: